### PR TITLE
Users can display numerical values for the wind speed

### PR DESCRIPTION
**Pull request by julienrf**
We introduce a new setting that allows users to display numerical values for the wind speed instead of the wind barbells. We show the numerical values everywhere (on the map overlays, on the meteograms, and on the sounding diagrams).

wind-numerical-values

### CODE REVIEW

File: frontend/src/LayersSelector.tsx [hunk 1]
```diff
@@ -86,7 +88,16 @@ export const LayersSelector = (props: {
     <input
       type='checkbox'
       checked={state.windLayerEnabled}
-      onChange={() => enableWindLayer(!state.windLayerEnabled)}
+      onChange={() => props.domain.enableWindLayer(!state.windLayerEnabled)}
+    />
+  );
+  const windNumericValuesCheckBox = inputWithLabel(
+    'Numerical values',
+    'Show numerical values instead of barbells',
```
Comment (Boran) [inline frontend/src/LayersSelector.tsx:96]: Not sur what a barbell is, how about "wind strength arrows" or "wind barbs"?

**Comment by Boran** [comment]
Nice User Interface improvement!

File: frontend/src/LayersSelector.tsx (hunk 1 above)
Comment (julienrf) [reply frontend/src/LayersSelector.tsx:96]: Good point, I badly translated “barbule”… I will fix that.

Comment (julienrf) [reply frontend/src/LayersSelector.tsx]: Done in ea42b92.

**Pull request merged**